### PR TITLE
updated examples to be real

### PR DESCRIPTION
**Pull request by aray-handy**

### CODE REVIEW

File: examples/pub.rb [hunk 1]
```diff
@@ -1,25 +1,59 @@
...
+              {"name" : "currency_code", "type" : "string"},
+              {"name" : "bus_object_id", "type" : "int"},
+              {"name" : "bus_object_type", "type" : "string"},
+              {"name" : "note", "type" : "string"},
+              {"name" : "type", "type" : "string"}
+             ]
+        }
+    }
+  ]
+}')
```
Comment (dsilfen-handy) [inline examples/pub.rb]: why do these as strings and not as a Ruby hash? This seems more error prone than it needs to be.

Comment (mtustin-handy) [inline examples/pub.rb]: This way you can see the exact json

**Comment by mtustin-handy** [comment]
👍 by my lights but I wouldn't say I know

File: examples/pub.rb (hunk 1 above)
Comment (dsilfen-handy) [inline examples/pub.rb]: It just seems like we should leverage the current language we are using to enforce that things are correct here at runtime. 

```ruby
schema_json = {
  type: :record
}.to_json
```

The above snippet seems more idiomatic to me.

Comment (aray-handy) [inline examples/pub.rb]: Good point. This was just an example for quick publishing of messages.

Comment (dsilfen-handy) [inline examples/pub.rb]: Also if you are going with a big string, might be nice to use HEREDOC syntax like we use for raw sql.

```ruby
raw_schema = <<-JSON
{
  "type": "record"
}
JSON
```

**Pull request merged**